pkg/scheduler/core/equivalence/eqivalence.go

```diff
@@ -97,7 +218,7 @@ type predicateResult struct {
 // run and its results cached for the next call.
 //
 // NOTE: RunPredicate will not update the equivalence cache if the given NodeInfo is stale.
-func (c *Cache) RunPredicate(
+func (c *NodeCache) RunPredicate(
```

Comment:
Similar to above, receiver variable should probably change from `c` to `n`.

Reply:
Fixed, thanks for catch this nits.